packages/core/src/ceramic.ts

```diff
@@ -170,20 +158,26 @@ class Ceramic implements CeramicApi {
     await dispatcher.init()
 
     const anchorServices = {
-      [IN_MEMORY_ANCHOR_SERVICE_CHAIN_ID]: [new InMemoryAnchorService(config)], // always included by default
+      [IN_MEMORY_CHAIN_ID]: [new InMemoryAnchorService(config)], // always included by default
     }
+    let preferredChain = IN_MEMORY_CHAIN_ID
 
     if (config.anchorServiceUrl) {
       const ethereumService = new EthereumAnchorService(config)
       const supportedChains = await ethereumService.getSupportedChains()
       for (const chainId of supportedChains) {
         anchorServices[chainId] = anchorServices[chainId] ? [...anchorServices[chainId], ethereumService] : [ethereumService]
       }
+      preferredChain = ROPSTEN_CHAIN_ID
     }
 
     const context: Context = {
       ipfs,
       anchorServices,
+      chainInfo: {
```

Comment:
Instead of putting this into the Context, I think it'd be cleaner to just put it into a private member of this Ceramic instance

Reply:
If you do that how would you call `makeGenesis` in the `TileDoctype` for example (`client` side)? You need to find out about the preferred chain, right? In that case you should request info by hitting the API for `getChainsInfo` and find out, is that right?

Reply:
Yep, exactly.  Anywhere you have access to a Context you have access to a CeramicAPI (and vice versa), so it doesn't actually change where this information is available, it's just changing the abstraction layer/separation of concerns